src/api.json

```diff
@@ -1339,8 +1339,11 @@
             },
             "csv_result": {
                 "description": "OK: The results for this request were retrieved successfully.",
+		"headers": {
+		    "content_disposition": "attachment"
+		},
                 "content": {
-                    "text/csv": {
+                    "application/vnd.ms-excel": {
```

Comment:
?

Reply:
FastAPI rituals from stackexchange - I don't think it actually makes a difference

Reply:
Can we change it back? It bothers me that the content type is incorrect.